Formula/stolon.rb

```diff
@@ -0,0 +1,33 @@
+$version = "v0.13.0"
+$bin_name = "stolon"
+
+class Stolon < Formula
+    desc "Stolon is a cloud native PostgreSQL manager for high availability"
+    homepage "https://github.com/sorintlab/stolon"
+    url "https://github.com/sorintlab/#{$bin_name}.git", tag: $version
+    version $version
+
+    depends_on "go" => :recommended
+    depends_on "postgresql" => :recommended
```

Comment:
This is a runtime requirement as far as I can tell, that means it's not just recommended

Reply:
As `postgresql` is needed for stolon at runtime, we removed `recommended`